### ISSUE TITLE
can not be installed on netbeans 8.2

### ISSUE THREAD

**Issue by jackiszhp** [Everlaw/nbts#93]
eventually, I am pretty sure the latest version of pluggin 2.1.5.3 can not be installed on netbeans 8.2. Since I have successfully installed it on netbeans 8.0.2. On 8.0.2, Yes, I can see it works as expected. but in 8.2, it can not even be installed successfully.

**Comment by arkhamvm** [comment]
Installed it on NB 8.2 via *.nbm without any problems.

**Comment by jackiszhp** [comment]
I do not know what is going on with me, after installation, it says restart IDE. But either "right now" or "later", after restart, i found that it is not installed.
Since I do not know how to solve this, I was even tried to build from source to find out, #94. And I found that 2.1.5.3 could not be built on netbeans 8.2.

All these made me frustrated, so I even went to try angularide, unfortunately it seems to require node 7. it will take me too much time to get node7, so I gave up.

Now I am using 8.0.2, and tolerating source files and dist files all together.
